jac-uk/admin
# #1314/additonal legal experience

## What's included?
Added a section under each item within legal experience containing `judicial-functions` which allows collected details to be displayed to admins. 

*Ive added some `<hr>` tags to add lines between items, this data is quite deeply nested and i felt the UI needed some separations*

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Find an 'legal' exercise 
![image](https://user-images.githubusercontent.com/44227249/118675271-5e7ab200-b7f2-11eb-808b-2213124856f2.png)
- Ensure an application has been made to this exercise, which has selected 'The carrying-out of judicial functions of any court or tribunal' under 'Post-qualification experience'. (apply side)
![image](https://user-images.githubusercontent.com/44227249/118675813-cf21ce80-b7f2-11eb-8d6d-26db20b4d2d9.png)
- Observe that the information entered is displayed on the admin portal.
![image](https://user-images.githubusercontent.com/44227249/118676116-0b552f00-b7f3-11eb-912b-47d058315dba.png)

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
![image](https://user-images.githubusercontent.com/44227249/118670995-01c9c800-b7ef-11eb-8493-95159177dc49.png)

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
